assets/src/application/ui/layout/espressoTable/ResponsiveTable.tsx

```diff
@@ -1,131 +1,85 @@
-// @ts-nocheck
 import React from 'react';
 import classNames from 'classnames';
 import uuidv4 from 'uuid/v4';
-import { find, isEmpty } from 'lodash'; // to be replaced with ramda
 
-/**
- * Internal dependencies
- */
+import { isEmpty } from '@appServices/utilities/array';
 import Table from './Table';
 import TableHeader from './TableHeader';
 import TableBody from './TableBody';
 import TableFooter from './TableFooter';
 import './style.scss';
 
-/**
- * ResponsiveDataGrid responsive-table
- * produces a table like structure for displaying tabular data
- * in a grid that collapses properly on smaller screens
- *
- */
-const ResponsiveTable = ({
-	headerRows = [],
-	tableRows = [],
+import { ResponsiveTableProps } from './types';
+
+const ResponsiveTable: React.FC<ResponsiveTableProps> = ({
+	className = {},
 	footerRows = [],
+	headerRows = [],
 	metaData = {},
-	classes = {},
 	onBeforeDragStart,
+	onDragEnd,
 	onDragStart,
 	onDragUpdate,
-	onDragEnd,
+	tableRows = [],
 	...props
-}: any) => {
-	const primaryHeader = find(headerRows, 'primary');
+}) => {
+	const primaryHeader = headerRows.find((row) => row.primary === true);
```

Comment:
yes, I tried to switch to ramda with something like:
```
const primaryHeader = R.find(R.propEq('primary', true)(headerRows);
```
but got some weird warning and since this is a pretty basic function, I've decided to go with raw JS, since ramda isn't helping a lot here.

The same reasoning was behind the `isEmpty` array check. Since it's a pretty basic check, ramda isn't adding much value.